CHANGELOG.next.md

```diff
@@ -18,6 +18,8 @@ Thanks, you're awesome :-) -->
 
 #### Improvements
 
+Updated description for 'syslog.severity.name' to clarify that the type is text-based. #2290
```

Comment:
I consider this a "bugfix" in the field's description. Can you move the changelog entry up under `Bugfixes`?